numpyro/contrib/render.py

```diff
@@ -1,385 +0,0 @@
-# Copyright Contributors to the Pyro project.
-# SPDX-License-Identifier: Apache-2.0
-
-import itertools
```

Comment:
Can we keep this file with a single line for backward-compatibility.
```
from numpyro.infer.inspect import render
```

Reply:
Done. importing the four functions here for full backwards compatibility.